CHANGELOG.md

```diff
@@ -1,4 +1,25 @@
-## 2.23.0 (Unreleased)
+## 2.24.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.9**
+- Supported Terraform version: **v1.x**
+
+### Features:
+#### Multi-Cloud Transit:
+1. Implemented support for exposing BGP LAN interface info on transit in Azure via ``bgp_lan_ip_list`` and ``ha_bgp_lan_ip_list``
+2. Implemented support for multiple disjoint port ranges for **aviatrix_microseg_policy_list**
+
+### Enhancements:
+1. Added support for updating ``bgp_md5_key`` and ``backup_bgp_md5_key`` for **aviatrix_spoke_external_device_conn** and **aviatrix_transit_external_device_conn**
+2. Added optimization on read operation for **aviatrix_transit_firenet_policy**
+
+### Bug Fixes:
+1. Fixed issue where NAT config has order issue and causes refresh problem for **aviatrix_gateway_dnat** and **aviatrix_gateway_snat**
```

Comment:
Fixed issue where NAT config had ordering issue and caused refresh problems for **aviatrix_gateway_dnat** and **aviatrix_gateway_snat**